### PR TITLE
Add/update API for region(s) infrastrucure

### PR DESCRIPTION
**Pull request by maretskiy**

### CODE REVIEW

File: raml/api.raml [hunk 1]
```diff
@@ -37,6 +37,19 @@ mediaType: application/json
             schema: !include schemas/200.json
             example: !include response_examples/200/regions_detailed.json
 
+  /regions/infra:
```
Comment (vkramskikh) [inline raml/api.raml]: I believe URL here should just be "/infra" (or "/api/v1/infra") to be homogenous with status, secutity issues, etc

**Review by vkramskikh — CHANGES_REQUESTED** [review]

File: raml/api.raml (hunk 1 above)
Comment (maretskiy) [reply raml/api.raml]: Agree, need to discuss this with @boris-42 since current URL is from the ticket

Comment (maretskiy) [reply raml/api.raml]: Renamed to /api/v1/infra

**Review by vkramskikh — APPROVED** [review]

**Review by boris-42 — APPROVED** [review]
it looks good overall. 

I would like to merge it, however could you please improve fake data in another PR (add situations with multiple URL per page)

**Pull request merged**